ios/FluentUI/Bottom Sheet/BottomCommandingController.swift

```diff
@@ -0,0 +1,592 @@
+                heroCommandStack.insertArrangedSubview(newView, at: stackIndex)
+            }
+        case .list:
+            if let cell = binding.view as? TableViewCell {
+                setupTableViewCell(cell, with: item)
+            }
+        }
+    }
+
+    private var itemToBindingMap: [CommandingItem: ItemBinding] = [:]
+
+    private var viewToBindingMap: [UIView: ItemBinding] = [:]
+
+    private var bottomBarView: UIView?
+
+    private var bottomSheetController: BottomSheetController?
+
+    private var isHeroCommandStackLoaded: Bool = false
+
+    private var isTableViewLoaded: Bool = false
+
+    private var isInSheetMode: Bool { bottomSheetController != nil }
+
+    private var isExpandable: Bool { expandedListSections.count > 0 }
+
+    private var bottomSheetHeroStackTopConstraint: NSLayoutConstraint?
+
+    private var bottomSheetHeroStackTopMargin: CGFloat {
+        isExpandable ? Constants.BottomSheet.heroStackExpandableTopMargin : Constants.BottomSheet.heroStackNonExpandableTopMargin
+    }
+
+    private var bottomSheetHeroStackHeight: CGFloat { Constants.heroButtonHeight + Constants.BottomSheet.heroStackBottomMargin + bottomSheetHeroStackTopMargin }
+
+    private var heroCommandWidthConstraints: [NSLayoutConstraint] {
+        heroItems.compactMap { (itemToBindingMap[$0] as? HeroItemBinding)?.widthConstraint }
+    }
+
+    private enum ItemLocation {
+        case heroSet
+        case list
+    }
+
+    private class ItemBinding {
+        let item: CommandingItem
+        let view: UIView
+        let location: ItemLocation
+
+        init(item: CommandingItem, view: UIView, location: ItemLocation) {
+            self.item = item
+            self.view = view
+            self.location = location
+        }
+    }
+
+    private class HeroItemBinding: ItemBinding {
+        let widthConstraint: NSLayoutConstraint
+
+        init(item: CommandingItem, view: UIView, location: ItemLocation, widthConstraint: NSLayoutConstraint) {
+            self.widthConstraint = widthConstraint
+            super.init(item: item, view: view, location: location)
+        }
+    }
+
+    private struct Constants {
+        static let heroButtonHeight: CGFloat = 48
+        static let heroButtonWidth: CGFloat = 96
+
+        static let tableViewIconTintColor: UIColor = Colors.textSecondary
+        static let tableViewBackgroundColor: UIColor = Colors.NavigationBar.background
+
+        struct BottomBar {
+            static let cornerRadius: CGFloat = 14
+            static let backgroundColor: UIColor = Colors.NavigationBar.background
+
+            static let bottomOffset: CGFloat = 30
+            static let heroStackLeadingTrailingMargin: CGFloat = 8
+            static let heroStackTopBottomMargin: CGFloat = 16
+
+            static let moreButtonIcon: UIImage? = UIImage.staticImageNamed("more-24x24")
+
+            // TODO: Localize
+            static let moreButtonTitle: String = "More"
```

Comment:
So it's not forgotten, I shall put a comment here :P

Reply:
Future PR :D This goes in the dev branch, not main yet.

Reply:
Ah, ok. If it was going into main, I would have requested a bug #. Cool!